analytics/analytics.py

```diff
@@ -1,12 +1,13 @@
 import streamlit as st
 from PIL import Image
 
-from pages import page_public_dataset
+from pages import page_public_dataset, page_tournesol_scores
 
 st.session_state.data = None
 
 PAGES = {
     "Public Dataset": page_public_dataset,
+    "Tournesol Scores": page_tournesol_scores,
```

Comment:
Very minor comment, feel free to ignore : )

I think we can be more explicit in the name of the page as they not only display different data, but also use different data source ( the first page load a file, the second directly connect to Tournesol with HTTPS ).

How about this:

```python
    "Load Public Dataset": page_public_dataset,
    "View Live Tournesol Scores": page_tournesol_scores,
```

or simply

```python
    "Load Public Dataset": page_public_dataset,
    "Live Tournesol Scores": page_tournesol_scores,
```

Reply:
I think I will change the Public Dataset page to download the dataset automatically  as it is done for the Tournesol score page.

Reply:
Now it is loaded automatically (and cached) as the score and the page is called "Comparisons Public Dataset"